native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -3251,7 +3241,8 @@ static int proxy_cluster_pre_request(proxy_worker **worker, proxy_balancer **bal
         check_workers(conf, r->server);
         node_storage->unlock_nodes();
         if (!(*balancer = ap_proxy_get_balancer(r->pool, conf, *url, 0))) {
-            ap_log_error(APLOG_MARK, APLOG_DEBUG, 0, r->server, "proxy: CLUSTER no balancer for %s", *url);
+            /* node_storage->unlock_nodes(); */
+            ap_log_error(APLOG_MARK, APLOG_ERR, 0, r->server, "proxy: CLUSTER no balancer for %s", *url);
```

Comment:
not sure here... Are we going to fill the log?

Reply:
I'll have a look.

Reply:
It seems ok, e.g., tests does not trigger it.